asab/library/providers/abc.py

```diff
@@ -2,13 +2,23 @@
 
 class LibraryProviderABC(object):
 
+	# List of extensions that are allowed to be present in the library
+	FileExtentions = {'.yaml', '.json'}
```

Comment:
this makes the library useless for the remote-control. I need at least `.conf`. `.yml` would be friendly also. Once there's a need to get any other file from the ZK, it is tough to debug why the library skips it especially as this list is hidden in the abc.

Reply:
Feel free to add other extensions - this is because it is quite problematic to differentiate folder and file in ZooKeeper.

Reply:
oh, I removed it from the ZK provider... :see_no_evil: when I ask the service to list me nodes from ZK, I expect a list of nodes from ZK. I can filter whatever I want afterwards. I always found this extension list problematic, but maybe I just can't see its beauty. :thinking:

Reply:
There is no beauty to it. Without this list, Zookeeper integration is way too slow.

Reply:
It is gone now.